### ISSUE TITLE
React v16 support

### ISSUE THREAD

**Issue by MayGo** [ctrlplusb/react-component-queries#95]
It works with v16,  but warnings about peer dependencies when installing.
Will https://github.com/ctrlplusb/react-component-queries/pull/92 be merged any time?

**Comment by hoschi** [comment]
can you make a release with that merge, please? @ctrlplusb

**Comment by ctrlplusb** [comment]
Yep, coming soon. Trying to prep a release of `react-sizeme` and I'll subsequently update and release `react-component-queries`. 👍
